.github/workflows/nightly.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         id: tag_version
         uses: mathieudutour/github-tag-action@v6.1
         with:
-          custom_tag: nightly
+          custom_tag: "nightly-${{ github.sha }}"
```

Comment:
great! can we do 'nightly--${{ github.sha }}' with single `'` to keep the pattern with `tag_prefix`?

Reply:
Of course meu patão